resources/lib/apihelper.py

```diff
@@ -93,24 +93,28 @@ def tvshow_to_listitem(self, tvshow, program, cache_file):
             context_menu=context_menu,
         )
 
-    def list_episodes(self, program=None, season=None, category=None, feature=None, programtype=None, page=None, use_favorites=False, variety=None):
+    def list_episodes(self, program=None, season=None, category=None, feature=None, programtype=None,
+                      page=None, items_per_page=None, use_favorites=False, variety=None, sort_key=None):
         """Construct a list of episode or season TitleItems from VRT NU Search API data and filtered by favorites"""
         # Caching
         if not variety:
             cache_file = None
-        elif use_favorites:
-            cache_file = 'my-{variety}-{page}.json'.format(variety=variety, page=page)
         else:
-            cache_file = '{variety}-{page}.json'.format(variety=variety, page=page)
+            cache_file = '{}{}{}.json'.format('my-' if use_favorites else '', variety, '-{}'.format(page) if sort_key is None else '')
```

Comment:
I understand what you are doing here, but I think it is becoming a bit clumsy.
I wonder if we could make it clearer by reformatting and still using kwargs.
```suggestion
            cache_file = '{my}{variety}{page}.json'.format(
                my='my-' if use_favorites else '',
                variety=variety,
                page='-{}'.format(page) if sort_key is None else '',
            )
```

Reply:
Thanks, this is better.